src/test/operators/export_binary_test.cpp

```diff
@@ -215,6 +236,33 @@ TEST_F(OperatorsExportBinaryTest, AllTypesDictionarySegment) {
   EXPECT_TRUE(compare_files("resources/test_data/bin/AllTypesDictionarySegment.bin", filename));
 }
 
+TEST_F(OperatorsExportBinaryTest, AllTypesRunLengthSegment) {
```

Comment:
We should probably use paramater templated tests here (see encoding tests). Or is there a good reason to not use them?
It's "only" a few lines right now, but will blow up once we have FoR, LZ4, ... tests for the binary export/import.